lightning/src/util/events.rs

```diff
@@ -249,6 +249,8 @@ pub enum Event {
 	/// [`ChannelManager::claim_funds`]: crate::ln::channelmanager::ChannelManager::claim_funds
 	/// [`ChannelManager::fail_htlc_backwards`]: crate::ln::channelmanager::ChannelManager::fail_htlc_backwards
 	PaymentReceived {
+		/// The node that recieved the payment, ie our node
```

Comment:
nit:
```suggestion
		/// The ID of the node that received the payment, i.e., our node.
```

Reply:
Is this field always going to be populated with "our" node id? 

I was wondering if there is merit in the "our" prefix here if that's the case.

Reply:
Good point, also the variable name should therefore maybe be called `receiver_node_id`?

Reply:
maybe I should drop the `i.e., our node` part of the comment as well as change the variable name to receiver_node_id?

Reply:
Yeah, seems reasonable.